template.yml

```diff
@@ -146,7 +146,7 @@ Resources:
       FunctionName: !Sub ${ResourcePrefix}-s3-getsignedurl
       CodeUri: ./functions/S3UploaderFunction/
       Handler: app.handler
-      Runtime: nodejs12.x
+      Runtime: nodejs16.x
```

Comment:
What testing has been done for this change?

Reply:
reverted back to v12